src/banana/cfg.py

```diff
@@ -1,3 +1,118 @@
 # -*- coding: utf-8 -*-
+"""Configurations related objects.
+
+This module contains the functions used to manage configurations.
+Moreover, the actual loaded configurations are stored in the module variable
+:data:`cfg`, that is used as the source of truth for them.
+
+"""
+import pathlib
+
+import appdirs
+import yaml
 
 cfg = None
+"Global store for configurations"
+
+default = {}
+
+name = "banana.yaml"
```

Comment:
Do we have a template file for how the `banana.yaml` should be?

Reply:
Not a proper one: it is only inside `eko` and `yadism` benchmarks. I'll provide one :)